[ 2014-02-25 20:46:53.1330 53935/0x7fff7b95e310 agents/Watchdog/Main.cpp:538 ]: Options: { 'analytics_log_user' => 'vlad', 'cleanup_pidfiles' => 'L3Zhci9mb2xkZXJzL3lyLzViXzIybWo5N2pxOGgweHdnajV6Zmt4dzAwMDBnbi9UL3Bhc3Nlbmdlci1zdGFuZGFsb25lLm0xMno2OS90ZW1wX2Rpcl90b3VjaGVyLnBpZA==', 'default_group' => 'staff', 'default_python' => 'python', 'default_ruby' => '/Users/vlad/.rvm/gems/ruby-2.1.1/wrappers/ruby', 'default_user' => 'vlad', 'log_level' => '0', 'max_pool_size' => '6', 'passenger_root' => '/var/folders/yr/5b_22mj97jq8h0xwgj5zfkxw0000gn/T/passenger-standalone.m12z69/locations.ini', 'passenger_version' => '4.0.37', 'pool_idle_time' => '300', 'prestart_urls' => 'aHR0cDovLzAuMC4wLjA6MzAwMAA=', 'temp_dir' => '/tmp', 'union_station_gateway_address' => 'gateway.unionstationapp.com', 'union_station_gateway_port' => '443', 'user_switching' => 'false', 'web_server_passenger_version' => '4.0.37', 'web_server_pid' => '53934', 'web_server_type' => 'nginx', 'web_server_worker_gid' => '-1', 'web_server_worker_uid' => '-1' }
[ 2014-02-25 20:46:53.1931 53937/0x7fff7b95e310 agents/HelperAgent/Main.cpp:643 ]: PassengerHelperAgent online, listening at unix:/tmp/passenger.1.0.53934/generation-0/request
[ 2014-02-25 20:46:53.3108 53939/0x7fff7b95e310 agents/LoggingAgent/Main.cpp:321 ]: PassengerLoggingAgent online, listening at unix:/tmp/passenger.1.0.53934/generation-0/logging
[ 2014-02-25 20:46:53.3111 53935/0x7fff7b95e310 agents/Watchdog/Main.cpp:728 ]: All Phusion Passenger agents started!
App 53954 stdout: 
App 53954 stdout: 
[ 2014-02-25 20:46:58.0493 53937/0x10ec4c000 Pool2/Implementation.cpp:883 ]: Could not spawn process for group /Developer/Sites/mehakovic#default: An error occured while starting up the preloader.
     in 'void Passenger::ApplicationPool2::SmartSpawner::handleErrorResponse(Passenger::ApplicationPool2::SmartSpawner::StartupDetails &)' (SmartSpawner.h:455)
     in 'string Passenger::ApplicationPool2::SmartSpawner::negotiatePreloaderStartup(Passenger::ApplicationPool2::SmartSpawner::StartupDetails &)' (SmartSpawner.h:566)
     in 'void Passenger::ApplicationPool2::SmartSpawner::startPreloader()' (SmartSpawner.h:206)
     in 'virtual ProcessPtr Passenger::ApplicationPool2::SmartSpawner::spawn(const Passenger::ApplicationPool2::Options &)' (SmartSpawner.h:752)
     in 'void Passenger::ApplicationPool2::Group::spawnThreadRealMain(const SpawnerPtr &, const Passenger::ApplicationPool2::Options &, unsigned int)' (Implementation.cpp:804)

[ 2014-02-25 20:46:58.0495 53937/0x10ebc6000 agents/HelperAgent/RequestHandler.h:2088 ]: [Client 21] Cannot checkout session.
Error page:
No such file or directory @ rb_sysopen - /Developer/Sites/mehakovic/_site/404.html (Errno::ENOENT)
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-contrib-1.1.0/lib/rack/contrib/not_found.rb:10:in `read'
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-contrib-1.1.0/lib/rack/contrib/not_found.rb:10:in `initialize'
  config.ru:9:in `new'
  config.ru:9:in `block in <main>'
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-1.5.2/lib/rack/builder.rb:55:in `instance_eval'
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-1.5.2/lib/rack/builder.rb:55:in `initialize'
  config.ru:1:in `new'
  config.ru:1:in `<main>'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:112:in `eval'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:112:in `preload_app'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:158:in `<module:App>'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:29:in `<module:PhusionPassenger>'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:28:in `<main>'

App 54097 stdout: 
App 54097 stdout: 
[ 2014-02-25 20:47:26.3959 53937/0x10ea5c000 Pool2/Implementation.cpp:883 ]: Could not spawn process for group /Developer/Sites/mehakovic#default: An error occured while starting up the preloader.
     in 'void Passenger::ApplicationPool2::SmartSpawner::handleErrorResponse(Passenger::ApplicationPool2::SmartSpawner::StartupDetails &)' (SmartSpawner.h:455)
     in 'string Passenger::ApplicationPool2::SmartSpawner::negotiatePreloaderStartup(Passenger::ApplicationPool2::SmartSpawner::StartupDetails &)' (SmartSpawner.h:566)
     in 'void Passenger::ApplicationPool2::SmartSpawner::startPreloader()' (SmartSpawner.h:206)
     in 'virtual ProcessPtr Passenger::ApplicationPool2::SmartSpawner::spawn(const Passenger::ApplicationPool2::Options &)' (SmartSpawner.h:752)
     in 'void Passenger::ApplicationPool2::Group::spawnThreadRealMain(const SpawnerPtr &, const Passenger::ApplicationPool2::Options &, unsigned int)' (Implementation.cpp:804)

[ 2014-02-25 20:47:26.3961 53937/0x10ebc6000 agents/HelperAgent/RequestHandler.h:2088 ]: [Client 21] Cannot checkout session.
Error page:
No such file or directory @ rb_sysopen - /Developer/Sites/mehakovic/_site/404.html (Errno::ENOENT)
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-contrib-1.1.0/lib/rack/contrib/not_found.rb:10:in `read'
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-contrib-1.1.0/lib/rack/contrib/not_found.rb:10:in `initialize'
  config.ru:9:in `new'
  config.ru:9:in `block in <main>'
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-1.5.2/lib/rack/builder.rb:55:in `instance_eval'
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-1.5.2/lib/rack/builder.rb:55:in `initialize'
  config.ru:1:in `new'
  config.ru:1:in `<main>'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:112:in `eval'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:112:in `preload_app'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:158:in `<module:App>'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:29:in `<module:PhusionPassenger>'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:28:in `<main>'

App 54239 stdout: 
App 54239 stdout: 
[ 2014-02-25 20:47:31.1722 53937/0x10ea5c000 Pool2/Implementation.cpp:883 ]: Could not spawn process for group /Developer/Sites/mehakovic#default: An error occured while starting up the preloader.
     in 'void Passenger::ApplicationPool2::SmartSpawner::handleErrorResponse(Passenger::ApplicationPool2::SmartSpawner::StartupDetails &)' (SmartSpawner.h:455)
     in 'string Passenger::ApplicationPool2::SmartSpawner::negotiatePreloaderStartup(Passenger::ApplicationPool2::SmartSpawner::StartupDetails &)' (SmartSpawner.h:566)
     in 'void Passenger::ApplicationPool2::SmartSpawner::startPreloader()' (SmartSpawner.h:206)
     in 'virtual ProcessPtr Passenger::ApplicationPool2::SmartSpawner::spawn(const Passenger::ApplicationPool2::Options &)' (SmartSpawner.h:752)
     in 'void Passenger::ApplicationPool2::Group::spawnThreadRealMain(const SpawnerPtr &, const Passenger::ApplicationPool2::Options &, unsigned int)' (Implementation.cpp:804)

[ 2014-02-25 20:47:31.1723 53937/0x10ebc6000 agents/HelperAgent/RequestHandler.h:2088 ]: [Client 21] Cannot checkout session.
Error page:
No such file or directory @ rb_sysopen - /Developer/Sites/mehakovic/_site/404.html (Errno::ENOENT)
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-contrib-1.1.0/lib/rack/contrib/not_found.rb:10:in `read'
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-contrib-1.1.0/lib/rack/contrib/not_found.rb:10:in `initialize'
  config.ru:9:in `new'
  config.ru:9:in `block in <main>'
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-1.5.2/lib/rack/builder.rb:55:in `instance_eval'
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-1.5.2/lib/rack/builder.rb:55:in `initialize'
  config.ru:1:in `new'
  config.ru:1:in `<main>'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:112:in `eval'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:112:in `preload_app'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:158:in `<module:App>'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:29:in `<module:PhusionPassenger>'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:28:in `<main>'

[ 2014-02-25 20:48:03.5790 54422/0x7fff7b95e310 agents/Watchdog/Main.cpp:538 ]: Options: { 'analytics_log_user' => 'vlad', 'cleanup_pidfiles' => 'L3Zhci9mb2xkZXJzL3lyLzViXzIybWo5N2pxOGgweHdnajV6Zmt4dzAwMDBnbi9UL3Bhc3Nlbmdlci1zdGFuZGFsb25lLml2NnNmZy90ZW1wX2Rpcl90b3VjaGVyLnBpZA==', 'default_group' => 'staff', 'default_python' => 'python', 'default_ruby' => '/Users/vlad/.rvm/gems/ruby-2.1.1/wrappers/ruby', 'default_user' => 'vlad', 'log_level' => '0', 'max_pool_size' => '6', 'passenger_root' => '/var/folders/yr/5b_22mj97jq8h0xwgj5zfkxw0000gn/T/passenger-standalone.iv6sfg/locations.ini', 'passenger_version' => '4.0.37', 'pool_idle_time' => '300', 'prestart_urls' => 'aHR0cDovLzAuMC4wLjA6MzAwMAA=', 'temp_dir' => '/tmp', 'union_station_gateway_address' => 'gateway.unionstationapp.com', 'union_station_gateway_port' => '443', 'user_switching' => 'false', 'web_server_passenger_version' => '4.0.37', 'web_server_pid' => '54421', 'web_server_type' => 'nginx', 'web_server_worker_gid' => '-1', 'web_server_worker_uid' => '-1' }
PassengerWatchdog(54425,0x7fff7b95e310) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerWatchdog(54425,0x7fff7b95e310) malloc: *** error: can't allocate region securely
PassengerWatchdog(54425,0x7fff7b95e310) malloc: *** set a breakpoint in malloc_error_break to debug
[ 2014-02-25 20:48:03.5958 54424/0x7fff7b95e310 agents/HelperAgent/Main.cpp:643 ]: PassengerHelperAgent online, listening at unix:/tmp/passenger.1.0.54421/generation-0/request
PassengerWatchdog(54427,0x7fff7b95e310) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerWatchdog(54427,0x7fff7b95e310) malloc: *** error: can't allocate region securely
PassengerWatchdog(54427,0x7fff7b95e310) malloc: *** set a breakpoint in malloc_error_break to debug
[ 2014-02-25 20:48:03.6070 54426/0x7fff7b95e310 agents/LoggingAgent/Main.cpp:321 ]: PassengerLoggingAgent online, listening at unix:/tmp/passenger.1.0.54421/generation-0/logging
[ 2014-02-25 20:48:03.6072 54422/0x7fff7b95e310 agents/Watchdog/Main.cpp:728 ]: All Phusion Passenger agents started!
App 54438 stdout: 
App 54438 stdout: 
[ 2014-02-25 20:48:06.3005 54424/0x102e6a000 Pool2/Implementation.cpp:883 ]: Could not spawn process for group /Developer/Sites/mehakovic#default: An error occured while starting up the preloader.
     in 'void Passenger::ApplicationPool2::SmartSpawner::handleErrorResponse(Passenger::ApplicationPool2::SmartSpawner::StartupDetails &)' (SmartSpawner.h:455)
     in 'string Passenger::ApplicationPool2::SmartSpawner::negotiatePreloaderStartup(Passenger::ApplicationPool2::SmartSpawner::StartupDetails &)' (SmartSpawner.h:566)
     in 'void Passenger::ApplicationPool2::SmartSpawner::startPreloader()' (SmartSpawner.h:206)
     in 'virtual ProcessPtr Passenger::ApplicationPool2::SmartSpawner::spawn(const Passenger::ApplicationPool2::Options &)' (SmartSpawner.h:752)
     in 'void Passenger::ApplicationPool2::Group::spawnThreadRealMain(const SpawnerPtr &, const Passenger::ApplicationPool2::Options &, unsigned int)' (Implementation.cpp:804)

[ 2014-02-25 20:48:06.3008 54424/0x102de4000 agents/HelperAgent/RequestHandler.h:2088 ]: [Client 29] Cannot checkout session.
Error page:
No such file or directory @ rb_sysopen - /Developer/Sites/mehakovic/_site/404.html (Errno::ENOENT)
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-contrib-1.1.0/lib/rack/contrib/not_found.rb:10:in `read'
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-contrib-1.1.0/lib/rack/contrib/not_found.rb:10:in `initialize'
  config.ru:9:in `new'
  config.ru:9:in `block in <main>'
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-1.5.2/lib/rack/builder.rb:55:in `instance_eval'
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-1.5.2/lib/rack/builder.rb:55:in `initialize'
  config.ru:1:in `new'
  config.ru:1:in `<main>'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:112:in `eval'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:112:in `preload_app'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:158:in `<module:App>'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:29:in `<module:PhusionPassenger>'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:28:in `<main>'

App 54580 stdout: 
App 54580 stdout: 
[ 2014-02-25 20:48:14.5670 54424/0x102c7a000 Pool2/Implementation.cpp:883 ]: Could not spawn process for group /Developer/Sites/mehakovic#default: An error occured while starting up the preloader.
     in 'void Passenger::ApplicationPool2::SmartSpawner::handleErrorResponse(Passenger::ApplicationPool2::SmartSpawner::StartupDetails &)' (SmartSpawner.h:455)
     in 'string Passenger::ApplicationPool2::SmartSpawner::negotiatePreloaderStartup(Passenger::ApplicationPool2::SmartSpawner::StartupDetails &)' (SmartSpawner.h:566)
     in 'void Passenger::ApplicationPool2::SmartSpawner::startPreloader()' (SmartSpawner.h:206)
     in 'virtual ProcessPtr Passenger::ApplicationPool2::SmartSpawner::spawn(const Passenger::ApplicationPool2::Options &)' (SmartSpawner.h:752)
     in 'void Passenger::ApplicationPool2::Group::spawnThreadRealMain(const SpawnerPtr &, const Passenger::ApplicationPool2::Options &, unsigned int)' (Implementation.cpp:804)

[ 2014-02-25 20:48:14.5671 54424/0x102de4000 agents/HelperAgent/RequestHandler.h:2088 ]: [Client 29] Cannot checkout session.
Error page:
No such file or directory @ rb_sysopen - /Developer/Sites/mehakovic/_site/404.html (Errno::ENOENT)
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-contrib-1.1.0/lib/rack/contrib/not_found.rb:10:in `read'
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-contrib-1.1.0/lib/rack/contrib/not_found.rb:10:in `initialize'
  config.ru:9:in `new'
  config.ru:9:in `block in <main>'
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-1.5.2/lib/rack/builder.rb:55:in `instance_eval'
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-1.5.2/lib/rack/builder.rb:55:in `initialize'
  config.ru:1:in `new'
  config.ru:1:in `<main>'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:112:in `eval'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:112:in `preload_app'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:158:in `<module:App>'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:29:in `<module:PhusionPassenger>'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:28:in `<main>'

App 54722 stdout: 
App 54722 stdout: 
[ 2014-02-25 20:48:16.5698 54424/0x102c7a000 Pool2/Implementation.cpp:883 ]: Could not spawn process for group /Developer/Sites/mehakovic#default: An error occured while starting up the preloader.
     in 'void Passenger::ApplicationPool2::SmartSpawner::handleErrorResponse(Passenger::ApplicationPool2::SmartSpawner::StartupDetails &)' (SmartSpawner.h:455)
     in 'string Passenger::ApplicationPool2::SmartSpawner::negotiatePreloaderStartup(Passenger::ApplicationPool2::SmartSpawner::StartupDetails &)' (SmartSpawner.h:566)
     in 'void Passenger::ApplicationPool2::SmartSpawner::startPreloader()' (SmartSpawner.h:206)
     in 'virtual ProcessPtr Passenger::ApplicationPool2::SmartSpawner::spawn(const Passenger::ApplicationPool2::Options &)' (SmartSpawner.h:752)
     in 'void Passenger::ApplicationPool2::Group::spawnThreadRealMain(const SpawnerPtr &, const Passenger::ApplicationPool2::Options &, unsigned int)' (Implementation.cpp:804)

[ 2014-02-25 20:48:16.5700 54424/0x102de4000 agents/HelperAgent/RequestHandler.h:2088 ]: [Client 29] Cannot checkout session.
Error page:
No such file or directory @ rb_sysopen - /Developer/Sites/mehakovic/_site/404.html (Errno::ENOENT)
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-contrib-1.1.0/lib/rack/contrib/not_found.rb:10:in `read'
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-contrib-1.1.0/lib/rack/contrib/not_found.rb:10:in `initialize'
  config.ru:9:in `new'
  config.ru:9:in `block in <main>'
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-1.5.2/lib/rack/builder.rb:55:in `instance_eval'
  /Users/vlad/.rvm/gems/ruby-2.1.1/gems/rack-1.5.2/lib/rack/builder.rb:55:in `initialize'
  config.ru:1:in `new'
  config.ru:1:in `<main>'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:112:in `eval'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:112:in `preload_app'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:158:in `<module:App>'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:29:in `<module:PhusionPassenger>'
  /usr/local/Cellar/passenger/4.0.37/libexec/helper-scripts/rack-preloader.rb:28:in `<main>'

[ 2014-02-25 20:48:56.2295 54886/0x7fff7b95e310 agents/Watchdog/Main.cpp:538 ]: Options: { 'analytics_log_user' => 'vlad', 'cleanup_pidfiles' => 'L3Zhci9mb2xkZXJzL3lyLzViXzIybWo5N2pxOGgweHdnajV6Zmt4dzAwMDBnbi9UL3Bhc3Nlbmdlci1zdGFuZGFsb25lLmczbWZjNC90ZW1wX2Rpcl90b3VjaGVyLnBpZA==', 'default_group' => 'staff', 'default_python' => 'python', 'default_ruby' => '/Users/vlad/.rvm/gems/ruby-2.1.1/wrappers/ruby', 'default_user' => 'vlad', 'log_level' => '0', 'max_pool_size' => '6', 'passenger_root' => '/var/folders/yr/5b_22mj97jq8h0xwgj5zfkxw0000gn/T/passenger-standalone.g3mfc4/locations.ini', 'passenger_version' => '4.0.37', 'pool_idle_time' => '300', 'prestart_urls' => 'aHR0cDovLzAuMC4wLjA6MzAwMAA=', 'temp_dir' => '/tmp', 'union_station_gateway_address' => 'gateway.unionstationapp.com', 'union_station_gateway_port' => '443', 'user_switching' => 'false', 'web_server_passenger_version' => '4.0.37', 'web_server_pid' => '54885', 'web_server_type' => 'nginx', 'web_server_worker_gid' => '-1', 'web_server_worker_uid' => '-1' }
PassengerWatchdog(54889,0x7fff7b95e310) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerWatchdog(54889,0x7fff7b95e310) malloc: *** error: can't allocate region securely
PassengerWatchdog(54889,0x7fff7b95e310) malloc: *** set a breakpoint in malloc_error_break to debug
[ 2014-02-25 20:48:56.2460 54888/0x7fff7b95e310 agents/HelperAgent/Main.cpp:643 ]: PassengerHelperAgent online, listening at unix:/tmp/passenger.1.0.54885/generation-0/request
[ 2014-02-25 20:48:56.2525 54890/0x7fff7b95e310 agents/LoggingAgent/Main.cpp:321 ]: PassengerLoggingAgent online, listening at unix:/tmp/passenger.1.0.54885/generation-0/logging
[ 2014-02-25 20:48:56.2527 54886/0x7fff7b95e310 agents/Watchdog/Main.cpp:728 ]: All Phusion Passenger agents started!
App 54911 stdout: 
[ 2014-02-25 20:48:58.9271 54888/0x10192b000 Pool2/SmartSpawner.h:301 ]: Preloader for /Developer/Sites/mehakovic started on PID 54911, listening on unix:/tmp/passenger.1.0.54885/generation-0/backends/preloader.54911
App 55053 stdout: 
PassengerWatchdog (cleaning up...)(55071,0x7fff7b95e310) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerWatchdog (cleaning up...)(55071,0x7fff7b95e310) malloc: *** error: can't allocate region securely
PassengerWatchdog (cleaning up...)(55071,0x7fff7b95e310) malloc: *** set a breakpoint in malloc_error_break to debug
[ 2014-02-25 20:49:51.2497 55109/0x7fff7b95e310 agents/Watchdog/Main.cpp:538 ]: Options: { 'analytics_log_user' => 'vlad', 'cleanup_pidfiles' => 'L3Zhci9mb2xkZXJzL3lyLzViXzIybWo5N2pxOGgweHdnajV6Zmt4dzAwMDBnbi9UL3Bhc3Nlbmdlci1zdGFuZGFsb25lLjEzb3pqZm4vdGVtcF9kaXJfdG91Y2hlci5waWQ=', 'default_group' => 'staff', 'default_python' => 'python', 'default_ruby' => '/Users/vlad/.rvm/gems/ruby-2.1.1/wrappers/ruby', 'default_user' => 'vlad', 'log_level' => '0', 'max_pool_size' => '6', 'passenger_root' => '/var/folders/yr/5b_22mj97jq8h0xwgj5zfkxw0000gn/T/passenger-standalone.13ozjfn/locations.ini', 'passenger_version' => '4.0.37', 'pool_idle_time' => '300', 'prestart_urls' => 'aHR0cDovLzAuMC4wLjA6MzAwMAA=', 'temp_dir' => '/tmp', 'union_station_gateway_address' => 'gateway.unionstationapp.com', 'union_station_gateway_port' => '443', 'user_switching' => 'false', 'web_server_passenger_version' => '4.0.37', 'web_server_pid' => '55108', 'web_server_type' => 'nginx', 'web_server_worker_gid' => '-1', 'web_server_worker_uid' => '-1' }
[ 2014-02-25 20:49:51.2685 55111/0x7fff7b95e310 agents/HelperAgent/Main.cpp:643 ]: PassengerHelperAgent online, listening at unix:/tmp/passenger.1.0.55108/generation-0/request
PassengerWatchdog(55114,0x7fff7b95e310) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerWatchdog(55114,0x7fff7b95e310) malloc: *** error: can't allocate region securely
PassengerWatchdog(55114,0x7fff7b95e310) malloc: *** set a breakpoint in malloc_error_break to debug
[ 2014-02-25 20:49:51.2755 55113/0x7fff7b95e310 agents/LoggingAgent/Main.cpp:321 ]: PassengerLoggingAgent online, listening at unix:/tmp/passenger.1.0.55108/generation-0/logging
[ 2014-02-25 20:49:51.2757 55109/0x7fff7b95e310 agents/Watchdog/Main.cpp:728 ]: All Phusion Passenger agents started!
App 55125 stdout: 
[ 2014-02-25 20:49:54.1250 55111/0x104a74000 Pool2/SmartSpawner.h:301 ]: Preloader for /Developer/Sites/mehakovic started on PID 55125, listening on unix:/tmp/passenger.1.0.55108/generation-0/backends/preloader.55125
App 55267 stdout: 
[ 2014-02-25 20:53:09.2595 55374/0x7fff7b95e310 agents/Watchdog/Main.cpp:538 ]: Options: { 'analytics_log_user' => 'vlad', 'cleanup_pidfiles' => 'L3Zhci9mb2xkZXJzL3lyLzViXzIybWo5N2pxOGgweHdnajV6Zmt4dzAwMDBnbi9UL3Bhc3Nlbmdlci1zdGFuZGFsb25lLjE4Ym4yczIvdGVtcF9kaXJfdG91Y2hlci5waWQ=', 'default_group' => 'staff', 'default_python' => 'python', 'default_ruby' => '/Users/vlad/.rvm/gems/ruby-2.1.1/wrappers/ruby', 'default_user' => 'vlad', 'log_level' => '0', 'max_pool_size' => '6', 'passenger_root' => '/var/folders/yr/5b_22mj97jq8h0xwgj5zfkxw0000gn/T/passenger-standalone.18bn2s2/locations.ini', 'passenger_version' => '4.0.37', 'pool_idle_time' => '300', 'prestart_urls' => 'aHR0cDovLzAuMC4wLjA6MzAwMAA=', 'temp_dir' => '/tmp', 'union_station_gateway_address' => 'gateway.unionstationapp.com', 'union_station_gateway_port' => '443', 'user_switching' => 'false', 'web_server_passenger_version' => '4.0.37', 'web_server_pid' => '55373', 'web_server_type' => 'nginx', 'web_server_worker_gid' => '-1', 'web_server_worker_uid' => '-1' }
[ 2014-02-25 20:53:09.2783 55376/0x7fff7b95e310 agents/HelperAgent/Main.cpp:643 ]: PassengerHelperAgent online, listening at unix:/tmp/passenger.1.0.55373/generation-0/request
[ 2014-02-25 20:53:09.2892 55378/0x7fff7b95e310 agents/LoggingAgent/Main.cpp:321 ]: PassengerLoggingAgent online, listening at unix:/tmp/passenger.1.0.55373/generation-0/logging
[ 2014-02-25 20:53:09.2894 55374/0x7fff7b95e310 agents/Watchdog/Main.cpp:728 ]: All Phusion Passenger agents started!
App 55391 stdout: 
[ 2014-02-25 20:53:12.1387 55376/0x107520000 Pool2/SmartSpawner.h:301 ]: Preloader for /Developer/Sites/mehakovic started on PID 55391, listening on unix:/tmp/passenger.1.0.55373/generation-0/backends/preloader.55391
App 55533 stdout: 
PassengerHelperAgent(55695,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(55695,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(55695,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(55703,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(55703,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(55703,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(55742,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(55742,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(55742,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(55829,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(55829,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(55829,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(55879,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(55879,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(55879,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(56133,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(56133,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(56133,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(56139,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(56139,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(56139,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(56257,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(56257,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(56257,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(56315,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(56315,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(56315,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(56317,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(56317,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(56317,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(56323,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(56323,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(56323,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(56336,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(56336,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(56336,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(56597,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(56597,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(56597,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(56623,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(56623,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(56623,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(56643,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(56643,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(56643,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(56684,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(56684,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(56684,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(56768,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(56768,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(56768,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(56806,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(56806,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(56806,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(56812,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(56812,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(56812,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(56869,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(56869,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(56869,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(56885,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(56885,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(56885,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(56938,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(56938,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(56938,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(57024,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(57024,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(57024,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(57030,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(57030,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(57030,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(57190,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(57190,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(57190,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(57214,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(57214,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(57214,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(57218,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(57218,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(57218,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(57240,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(57240,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(57240,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(57268,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(57268,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(57268,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(57290,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(57290,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(57290,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(57329,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(57329,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(57329,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(57355,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(57355,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(57355,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(57359,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(57359,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(57359,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(57367,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(57367,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(57367,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(57422,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(57422,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(57422,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(57432,0x1072aa000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(57432,0x1072aa000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(57432,0x1072aa000) malloc: *** set a breakpoint in malloc_error_break to debug
[ 2014-02-25 21:58:53.2603 55378/0x7fff7b95e310 agents/LoggingAgent/Main.cpp:289 ]: Caught signal, exiting...
[ 2014-03-17 21:19:56.0436 1192/0x7fff7bd33310 agents/Watchdog/Main.cpp:538 ]: Options: { 'analytics_log_user' => 'vlad', 'cleanup_pidfiles' => 'L3Zhci9mb2xkZXJzL3lyLzViXzIybWo5N2pxOGgweHdnajV6Zmt4dzAwMDBnbi9UL3Bhc3Nlbmdlci1zdGFuZGFsb25lLjFwM2wzaW8vdGVtcF9kaXJfdG91Y2hlci5waWQ=', 'default_group' => 'staff', 'default_python' => 'python', 'default_ruby' => '/Users/vlad/.rvm/gems/ruby-2.1.1/wrappers/ruby', 'default_user' => 'vlad', 'log_level' => '0', 'max_pool_size' => '6', 'passenger_root' => '/var/folders/yr/5b_22mj97jq8h0xwgj5zfkxw0000gn/T/passenger-standalone.1p3l3io/locations.ini', 'passenger_version' => '4.0.37', 'pool_idle_time' => '300', 'prestart_urls' => 'aHR0cDovLzAuMC4wLjA6MzAwMAA=', 'temp_dir' => '/tmp', 'union_station_gateway_address' => 'gateway.unionstationapp.com', 'union_station_gateway_port' => '443', 'user_switching' => 'false', 'web_server_passenger_version' => '4.0.37', 'web_server_pid' => '1191', 'web_server_type' => 'nginx', 'web_server_worker_gid' => '-1', 'web_server_worker_uid' => '-1' }
[ 2014-03-17 21:19:57.2028 1194/0x7fff7bd33310 agents/HelperAgent/Main.cpp:643 ]: PassengerHelperAgent online, listening at unix:/tmp/passenger.1.0.1191/generation-0/request
PassengerWatchdog(1197,0x7fff7bd33310) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerWatchdog(1197,0x7fff7bd33310) malloc: *** error: can't allocate region securely
PassengerWatchdog(1197,0x7fff7bd33310) malloc: *** set a breakpoint in malloc_error_break to debug
[ 2014-03-17 21:19:57.4702 1196/0x7fff7bd33310 agents/LoggingAgent/Main.cpp:321 ]: PassengerLoggingAgent online, listening at unix:/tmp/passenger.1.0.1191/generation-0/logging
[ 2014-03-17 21:19:57.4704 1192/0x7fff7bd33310 agents/Watchdog/Main.cpp:728 ]: All Phusion Passenger agents started!
App 1208 stdout: 
[ 2014-03-17 21:20:14.4289 1194/0x10efa4000 Pool2/SmartSpawner.h:301 ]: Preloader for /Developer/Sites/mehakovic started on PID 1208, listening on unix:/tmp/passenger.1.0.1191/generation-0/backends/preloader.1208
App 1352 stdout: 
App 1353 stdout: 
PassengerHelperAgent(2420,0x10ed2e000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(2420,0x10ed2e000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(2420,0x10ed2e000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerHelperAgent(2422,0x10ed2e000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(2422,0x10ed2e000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(2422,0x10ed2e000) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerWatchdog (cleaning up...)(19128,0x7fff7bd33310) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerWatchdog (cleaning up...)(19128,0x7fff7bd33310) malloc: *** error: can't allocate region securely
PassengerWatchdog (cleaning up...)(19128,0x7fff7bd33310) malloc: *** set a breakpoint in malloc_error_break to debug
PassengerWatchdog (cleaning up...)(19132,0x7fff7bd33310) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerWatchdog (cleaning up...)(19132,0x7fff7bd33310) malloc: *** error: can't allocate region securely
PassengerWatchdog (cleaning up...)(19132,0x7fff7bd33310) malloc: *** set a breakpoint in malloc_error_break to debug
